.storybook/preview.js

```diff
@@ -20,6 +20,7 @@ addParameters({
 			return a[1].kind === b[1].kind ? 0 : a[1].id.localeCompare(b[1].id, undefined, { numeric: true });
 		},
 		showRoots: true,
+		enableShortcuts: false,
```

Comment:
Sneaking this in because Storybook was driving me crazy showing/hiding panels because of keystrokes.

Reply:
thanks!